data/community/functions/favor/clear_weather/activate.mcfunction

```diff
@@ -3,7 +3,7 @@ clear @s minecraft:ghast_tear{display:{Name:'{"italic":false,"translate":"item.m
 
 # run commands
 weather clear
-gamerule doWeatherCycle false
+gamerule doWeatherCycle true
```

Comment:
I think you changed this in the wrong file...

`doWeatherCycle` should be kept as `false` in the activate function, so that the weather cycle stops when the favor is activated.

And it should be set back to `true` in the deactivate function, so that the weather cycle resumes when the favor deactivates.

Reply:
That is right, I wasn't paying attention to which file I was changing it to.